p2panda-net-next/src/actors/events.rs

```diff
@@ -1,6 +1,9 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
 //! Events actor.
+//!
+//! Receives events from other actors, aggregating and enriching them before informing
```

Comment:
Just a note for the future on composing events from what I've observed doing it in p2panda-spaces. Rather than the actors themselves composing events and sending them "ready formed" to the events actor, it might be a nicer pattern to send `prev_state` and `next_state` objects, along with maybe any message which was just processed, and then have the events actor itself composing the events, and optionally adding any aggregate data as well. 

Would keep all the event logic nicely in one place, instead of repeating a lot in all actors.